pkg/scheduler/plugins/predicates/predicates.go

```diff
@@ -376,7 +375,14 @@ func (pp *predicatesPlugin) OnSessionOpen(ssn *framework.Session) {
 		if predicate.podAffinityEnable {
 			_, status := podAffinityFilter.PreFilter(context.TODO(), state, task.Pod)
 			if status.IsSkip() {
-				state.SkipFilterPlugins.Insert(interpodaffinity.Name)
+				taskKey := api.PodKey(task.Pod)
+				if _, ok := skipPlugins[taskKey]; !ok {
+					plugins := sets.New[string]()
+					skipPlugins[taskKey] = plugins
+				}
+				skipPlugins[taskKey].Insert(interpodaffinity.Name)
+				klog.V(3).Infof("pod(%s/%s) affinity require information is nil, plugin InterPodAffinity is skip",
```

Comment:
change "pod(%s/%s) affinity require information is nil, plugin InterPodAffinity is skip" to "pod(%s/%s) affinity require information is nil, plugin InterPodAffinity is skipped."

Reply:
done